.circleci/config.yml

```diff
@@ -1,26 +1,39 @@
 version: 2
+
 jobs:
   build:
     docker:
-      - image: circleci/node:8
+      - image: circleci/node:10
 
     working_directory: ~/repo
 
     steps:
       - checkout
 
-      # Download and cache dependencies
       - restore_cache:
           keys:
-          - v1-dependencies-{{ checksum "package.json" }}
-          # fallback to using the latest cache if no exact match is found
-          - v1-dependencies-
+            - npm-{{ arch }}-{{ .Branch }}-{{ checksum "package-lock.json" }}
+            - npm-{{ arch }}-{{ .Branch }}-{{ checksum "package-lock.json" }}
```

Comment:
Is `Branch` necessary? Seems like it'll just limit cache reuse.

Reply:
No, man is CircleCI complex.